support-models/src/main/scala/com/gu/support/workers/ProductTypeRatePlans.scala

```diff
@@ -1,70 +1,54 @@
 package com.gu.support.workers
 
+import com.gu.support.catalog
 import com.gu.support.catalog.{Product, ProductRatePlan}
 import com.gu.support.config.TouchPointEnvironment
 import com.gu.support.zuora.api.ReaderType
 import com.gu.support.zuora.api.ReaderType.Corporate
 
-
-trait ProductTypeRatePlans[P] {
```

Comment:
there seemed to be a lot of boilerplate and implicits here, for something that was only used in 4 places, and each of those could just be a normal function call.